internal/common/gamestate/gamestate.go

```diff
@@ -43,6 +36,7 @@ func (g GameState) Copy() GameState {
 	ret.ClientInfos = copyClientInfos(g.ClientInfos)
 	ret.Environment = g.Environment.Copy()
 	ret.DeerPopulation = g.DeerPopulation.Copy()
+	ret.IIGOHistory = g.IIGOHistory
```

Comment:
this does _not_ copy.
Please make `IIGOHistory` a non pointer and do [slice copying](https://www.geeksforgeeks.org/how-to-copy-one-slice-into-another-slice-in-golang/).

Currently, logs show:
```
2020-12-26T20:39:54.9132181Z ===== START OF TURN 1 (END OF TURN 0) =====
2020-12-26T20:39:54.9144200Z gamestate.GameState{Season:0x1, Turn:0x1, CommonPool:0, ClientInfos:map[shared.ClientID]gamestate.ClientInfo{Team1:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}, Team2:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}, Team3:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}, Team4:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}, Team5:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}, Team6:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}, Environment:disasters.Environment{Geography:disasters.ArchipelagoGeography{Islands:map[shared.ClientID]disasters.IslandLocationInfo{Team1:disasters.IslandLocationInfo{ID:Team1, X:3, Y:0}, Team2:disasters.IslandLocationInfo{ID:Team2, X:4, Y:0}, Team3:disasters.IslandLocationInfo{ID:Team3, X:5, Y:0}, Team4:disasters.IslandLocationInfo{ID:Team4, X:0, Y:0}, Team5:disasters.IslandLocationInfo{ID:Team5, X:1, Y:0}, Team6:disasters.IslandLocationInfo{ID:Team6, X:2, Y:0}}, XMin:0, XMax:10, YMin:0, YMax:0}, DisasterParams:disasters.disasterParameters{globalProb:0.1, spatialPDF:0, magnitudeLambda:1}, LastDisasterReport:disasters.DisasterReport{Magnitude:0, X:0, Y:0}}, DeerPopulation:foraging.DeerPopulationModel{deProblem:simulation.ODEProblem{YPrime:(simulation.ypFunc)(0x4e1120), T0:0, Y0:12, DtStep:0.1}, Population:12, T:0}, IIGOHistory:(*[]shared.Accountability)(0x65cfc0)}
```

And if you look at the field in `output.json`, you will find that it is the _same_ for every turn. Because you are modifying essentially the same slice in memory every turn, and not making a _deep copy_.